docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are
+primarily a concern of the *driver* layer (see [Architecture]({{< relref "architecture.md" >}})). This means that the
+specifics of authentication could differ based on the Fluid service and its driver.
```

Comment:
I think it's clearer to specific to the service instead of the driver. I could imagine a deep dive doc here describing how auth works relative to the layers. From the consumers point of view they have a service-client package and we are only making them responsible for that.

 ```suggestion
Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are implemented based on the Fluid service it's connecting to. This means that the specifics of authentication will differ based on the Fluid service.
```